modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/common/FieldBase.tsx

```diff
@@ -121,6 +121,7 @@ export default function FieldBase({
 				<FieldFeedback
 					errorMessage={errorMessage}
 					helpMessage={helpMessage}
+					id={`${id}fieldFeedback`}
```

Comment:
Why do we need to set `id`?

Reply:
We need the `id` here so the screen reader can make a relation to the `aria-describedby` and give a proper description of the component.

Reply:
Got it, thanks for the clarification!